lib/linguist/languages.yml

```diff
@@ -1975,6 +1975,7 @@ JSX:
   type: programming
   group: JavaScript
   extensions:
+  - ".js"
```

Comment:
Is `.js` really the primary extension for JSX? If not, please move this below `.jsx` as the first extension is considered the primary extension.